resource/utilities/command.cpp

```diff
@@ -286,6 +291,75 @@ static int update (std::shared_ptr<resource_context_t> &ctx,
     return update_run (ctx, args[2], buffer.str (), jobid, at, d);
 }
 
+static int emit_vtx_edg (std::shared_ptr<resource_context_t> &ctx, 
+                         std::stringstream &o, const std::string &status)
+{
+    int rc = -1;
+    f_vtx_iterator_t vi, v_end;
+    f_edg_iterator_t ei, e_end;
+    f_resource_graph_t fg = *(ctx->fgraph);
+    std::unordered_set<vtx_t> vtx_set;
+    resource_pool_t::string_to_status sts = resource_pool_t::str_to_status;
+    
+    if (status == "ANY") {
+        for (tie (vi, v_end) = vertices (fg); vi != v_end; ++vi) {
+            if (ctx->writers->emit_vtx ("", fg, *vi, 
+                  fg[*vi].size, false) < 0) {
+                std::cerr << "ERROR: writer emit vertex: " 
+                            << strerror (errno) << std::endl;
+                goto done;
+            }
+            auto ret = vtx_set.insert (*vi);
+            if (!ret.second) {
+                std::cerr << "ERROR: detected a duplicate vertex" << std::endl;
+                goto done;
+            }
+        }
+    }
+    else {
+        auto status_it = sts.find (status);
+        if (status_it == sts.end ()) {
+            std::cerr << "ERROR: unrecognized status" << std::endl;
+            goto done;
+        } 
+        else {
+            for (tie (vi, v_end) = vertices (fg); vi != v_end; ++vi) {
+                if (fg[*vi].status == status_it->second) {
+                    if (ctx->writers->emit_vtx ("", fg, *vi, 
+                           fg[*vi].size, false) < 0) {
+                        std::cerr << "ERROR: writer emit vertex: " 
+                                    << strerror (errno) << std::endl;
+                        goto done;
+                    }
+                    auto ret = vtx_set.insert (*vi);
+                    if (!ret.second) {
+                        std::cerr << "ERROR: detected a duplicate vertex"
+                                     << std::endl;
+                        goto done;
+                    }
+                }
+            }
+        }
+    }
+
+    for (tie (ei, e_end) = edges (fg); ei != e_end; ++ei) {
+        // count () returns 1 or 0 for existence or absence, respectively 
+        // of the item in the unordered set.  It is an O(1) operation.
+        if ( (vtx_set.count (target (*ei, fg)) == 1) 
+                && (vtx_set.count (source (*ei, fg)) == 1) ) {
+            if (ctx->writers->emit_edg ("", fg, *ei) < 0) {
+                std::cerr << "ERROR: writer emit edge: " 
+                            << strerror (errno) << std::endl;
+                goto done;
+            }
+        }
+    }
+
+    rc = 0;
+done:
+    return 0;
```

Comment:
```suggestion
    return rc;
```

When I compiled, my compiler warned that:
```
command.cpp: In function ‘int Flux::resource_model::emit_vtx_edg(std::shared_ptr<Flux::resource_model::resource_context_t>&, std::stringstream&, const string&)’:
command.cpp:297:9: warning: variable ‘rc’ set but not used [-Wunused-but-set-variable]
     int rc = -1;
```

Reply:
Good catch!

Reply:
Actually I want to return 0 to return REPL control, so `return 0` is correct, but there's not need for `rc`.